### PR TITLE
Tidy up the spec

### PR DESCRIPTION
**Pull request by staltz**
Making this ready for being published as an official milestone doc.

I recommend reviewing this PR by looking at the "markdown rich diff": https://github.com/ssbc/ssb-meta-feed-group-spec/pull/7/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

### CODE REVIEW

**Review by arj03 — APPROVED** [review]
Looks good to me. The sympathy could be more specific (like if the group is encrypted, then we probably don't know much more than here is an encrypted feed of some kind, maybe its a group), but I think overall this is a nice improvement.

**Comment by staltz** [comment]
Thanks for taking a look!

Yeah, let's leave sympathy for later and make a "v1.1" for this spec. We gotta ship this milestone.

Waiting for @mixmix's review.

File: README.md [hunk 1]
```diff
@@ -63,56 +86,49 @@ This feed MUST be unique for each peer (a singleton).
...
+We choose the the group `secret` because it is a value known only to those already in the group.
+  </div>
+</details>
+    
+<details>
+  <summary>Details about the group feed</summary>
+  <div>
+
+* `feedpurpose = secret` where `secret` is the base64 encoded group secret
+* `feedFormat = classic`
```
Comment (mixmix) [inline README.md:121]:
```suggestion
* `feedformat = classic`
```

**Review by mixmix — APPROVED** [review]
Looks good.
I have some changes I wrote up but will re-work those to line up after this.

What's still needed is the DM part of the spec (for `group/add-member` messages)... that's what I was working on.

**Pull request merged**